libpromises/evalfunction.c

```diff
@@ -1187,7 +1187,7 @@ static FnCallResult FnCallConcat(EvalContext *ctx, ARG_UNUSED const Policy *poli
 
     snprintf(id, CF_BUFSIZE, "built-in FnCall concat-arg");
 
-/* We need to check all the arguments, ArgTemplate does not check varadic functions */
+    /* We need to check all the arguments, ArgTemplate does not check varadic functions */
```

Comment:
Unrelated change, but ok.

Reply:
All right, **now** it is unrelated. Previously it was just next to the function I wrote, so I had to fix while I was on it. I moved the function now to be close to other string_* so it became unrelated...

Reply:
I reverted this now.